### DISCUSSION TITLE
I may have found an exploit to install any extension on a managed account even with allowlist

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1127] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
i need somebody to test this, if you use another chromebook thats on a version that extension installer works and unenrolled and your school email has sync turned on, then you put it in dev mode, then you login to school email, then you open crosh and use pollen policy editor to remove allowlist and any other blocks then use the extension installer to install any extension you like, then it should install on your enrolled chromebook correct when you open up your school chromebook?

**Comment by AshtonDavies** [comment]
No one wants to use such a lengthy process like that. Use SKIOVOX if you can and install extensions that way.

**Reply by Balistic123** [reply]
for people that dont have skiovox or cant use skiovox

**Reply by Mazedotexe** [reply]
> for people that dont have skiovox or cant use skiovox

like me
(ofc I don't have another Chromebook tho)

**Reply by Balistic123** [reply]
you can get a really cheap one for like $10-$15 on ebay I can give you a couple of links

**Reply by minnow-bible** [reply]
I tried this and it most likely will not work. The policy is only overwritten for the secondary chromebook with dev mode enabled. I tried using the modified chromebook to install extensions to my account, but unfortunately it is too old to use the webstore. Regardless, even if it had been newer, I believe the allow list policy would uninstall whatever extension you use the modified chromebook to install. 

In two or three months I will test with a newer, web-store compatible chromebook, but you should expect the same negative results.

**Comment by raahilaalam** [comment]
it does not work.crosh is disabled,even if sync is off,the only way
